hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,464 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Generated UNR file from Synopsys UNR tool with security modules being black-boxed.
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Aug 11 14:09:51 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div2_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div2_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div4_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div4_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_aes_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
```

Comment:
hmmm...going to need some help here again..what condition does it think it's not possible..? `wr_en == 1`?

Reply:
Right, the hint_status CSR are read-only, so we is tied to 0, and wr_en is never active.